### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nJobes**

### CODE REVIEW

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class StudentController {
 	
 	@PostMapping("/student/add")
 	@Transactional
-	public StudentDTO addCourse( @RequestBody StudentDTO studentDTO  ) { 
+	public StudentDTO addStudent( @RequestBody StudentDTO studentDTO  ) {
```
Comment (jeromeportega) [inline cst438_register/src/main/java/com/cst438/controller/StudentController.java:33]: How come the method name change?

File: cst438_register/src/test/java/com/cst438/EndToEndAddStudentTest.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
+package com.cst438;
```
Comment (jeromeportega) [inline cst438_register/src/test/java/com/cst438/EndToEndAddStudentTest.java:1]: Really great testing.  Very straightforward and well commented.  Fantastic PR.

**Review by jeromeportega — APPROVED** [review]

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java (hunk 1 above)
Comment (nJobes) [reply cst438_register/src/main/java/com/cst438/controller/StudentController.java:33]: Just a forgotten artifact from copy pasta, didn't notice it until now.

File: cst438_register/src/test/java/com/cst438/EndToEndAddStudentTest.java (hunk 2 above)
Comment (nJobes) [reply cst438_register/src/test/java/com/cst438/EndToEndAddStudentTest.java:1]: Thanks.

**Pull request merged**